app/frontend/src/App.tsx

```diff
@@ -14,13 +17,17 @@ function App() {
   return (
     <Router>
       <ThemeProvider theme={theme}>
-        <ReactQueryClientProvider>
-          <AuthProvider>
-            <CssBaseline />
-            <EnvironmentBanner />
-            <AppRoutes />
-          </AuthProvider>
-        </ReactQueryClientProvider>
+        <FatalErrorBoundary fallbackRender={() => <ErrorFallback isFatal />}>
+          <ReactQueryClientProvider>
+            <AuthProvider>
+              <ErrorBoundary>
```

Comment:
Why this one as well as the fatal one? It also seems a bit odd to have our component `ErrorBoundary` but also use the one from the package directly.